apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -414,9 +415,9 @@ handle_request('PostSpend', #{'SpendTx' := Req}, _Context) ->
                                        {recipient_pubkey, recipient},
                                         amount, fee, payload]),
                  read_optional_params([{ttl, ttl, '$no_value'}]),
-                 base58_decode([{sender, sender, account_pubkey},
-                                {recipient, recipient, account_pubkey}]),
-                 get_nonce(sender),
+                 base58_decode([{sender, sender, id_hash},
```

Comment:
For anyone reviewing this commit in isolation from the rest of this PR - this change seems to relax check on param though this is made again as tight in [this following commit](https://github.com/aeternity/epoch/pull/1332/commits/767265eec4b14bc21b8aeade1873e70f58ce3d8a#diff-453a86322d2489c390db7fde31659cceR417).